tests/netstats_l2/tests/01-run.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Alexandre Abadie <alexandre.abadie@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+EXPECTED_HELP = (
+    'Command              Description',
+    '---------------------------------------',
+    'reboot               Reboot the node',
+    'ifconfig             Configure network interfaces',
+    '>'
+)
+
+EXPECTED_IFCONFIG = [
+    r'ifconfig',
+    (r'Iface  \d+   HWaddr\: ([0-9a-f]{2}:){1,}([0-9a-f]{2}) '
+     r'(Channel\: [11-26]  )?(Page\: \d+  )?(NID\: 0x[0-9a-f]{2})?')
+]
+
+if os.environ['BOARD'] != 'native':
+    EXPECTED_IFCONFIG += [
+        r'           Long HWaddr\: ([0-9a-f]{2}:){7}([0-9a-f]{2})',
+        r'           TX-Power\: (-?\d+)dBm  State\: IDLE  max\. Retrans\.\: \d+  CSMA Retries\: \d+',
+        r'           ACK\_REQ  CSMA',
+    ]
+
+EXPECTED_IFCONFIG += [
+    r'       Source address length: \d+',
+    r'       Statistics for Layer 2',
+    r'        RX packets \d+  bytes \d+',
+    r'        TX packets \d+ \(Multicast: \d+\)  bytes \d+',
+    r'        TX succeeded \d+ errors 0'
```

Comment:
Shouldn't it suffice to just expect the 4 lines above on `ifconfig`?

Reply:
That was the first version of the test. I can revert if you prefer.

Reply:
What were the reasons you changed it?

Reply:
I initially wrote it and tested it for native because I had no netif board with me. But then I realized the output was more complex with samr21-xpro so I tried to have a more generic version of the test (improving my regexp skills in the meantime ;) )

Reply:
Well, but the version you have now is not more generic, but more specific... so I would prefer if you revert to your more generic approach ;-).